channels-and-providers/sms/clickatell.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Clickatell'
+description: 'Learn how to use the Clickatell provider to send sms notifications using Novu'
+---
+
+You can use the [Clickatell](https://www.clickatell.com/) provider to send SMS messages to your customers using the Novu Platform with a single API to create multi-channel experiences.
```

Comment:
I don't understand the '...with single API to create multi-channel experiences' part. Is it about Novu?

If yes, I recommend removing it entirely. If someone is on this page, they already know what Novu can do and are interested in getting it to work with Clickatell so the focus should be on that.